src/main/resources/db_scripts/db_script_latest.sql

```diff
@@ -97,12 +97,15 @@ create table "job_template" (
   "key_value_pairs" VARCHAR NOT NULL,
   "id" BIGSERIAL NOT NULL PRIMARY KEY,
   "form_config" VARCHAR NOT NULL DEFAULT 'unknown'
+<<<<<<< HEAD:src/main/resources/db_scripts/db_script_v8.sql
+=======
```

Comment:
Please resolve your diffs 🤣

Reply:
I see you resolved this, but the diff is still here. Did you forget to commit some changes?